ui/src/apps/clusterInfo/components/ClusterInfoTable.js

```diff
@@ -31,9 +30,47 @@ function ComponentPanelTable({ cluster }) {
       key: 'deploy_path',
       ellipsis: true,
     },
+    {
+      title: t('cluster_info.component_table.status'),
+      dataIndex: 'status',
+      key: 'status',
+      width: 100,
+      render: (text, node) => {
+        if (node && node.comp && node.comp === 'tidb') {
+          let showConfirm = () => {
+            Modal.confirm({
+              title: t('cluster_info.component_table.hide_db'),
+              content: t('cluster_info.component_table.hide_warning'),
+              onOk() {
+                client.dashboard.topologyTidbAddressDelete(node.address)
+              },
+              onCancel() {},
+            });
+          }
+          return (
+            <span>
+              <span>{text} </span>
```

Comment:
i18n?

Reply:
`{text} ` is already an i18n string when it is used as the argument in `render`